src/Views/FirmwareView.vala

```diff
@@ -120,12 +120,12 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
                 add_device (devices[i]);
             }
 
-            placeholder_alert_view.title = _("Firmware Updates Are Not Available");
-            placeholder_alert_view.description = _("Firmware updates are not supported on this or any connected devices.");
+            placeholder_alert_view.title = _("No new firmware found");
+            placeholder_alert_view.description = _("Nothing on or connected to your system supports installing new firmware this way.");
             update_list.show_all ();
         } catch (Error e) {
-            placeholder_alert_view.title = _("The Firmware Service Is Not Available");
-            placeholder_alert_view.description = _("Please make sure “fwupd” is installed and enabled.");
```

Comment:
Shouldn't be "enabled", because the distinction to "turned on/off" is not meaningful here. (extra complexity invoked).
What "enabled" even is isn't explained. This means the user doesn't get to know what "fwupd" is.